helm/zot/templates/pvc.yaml

```diff
@@ -13,3 +13,30 @@ spec:
   storageClassName: {{ .Values.pvc.storageClassName }}
   {{- end }}
 {{- end }}
+
+{{- if or (.Capabilities.APIVersions.Has "kyverno.io/v2alpha1/PolicyException") (eq .Values.pvc.policyException.enforce true) }}
+apiVersion: kyverno.io/v2alpha1
+kind: PolicyException
+metadata:
+  name: allow-pvc-{{ .Release.Name | quote }}
+  {{- if (.Values.pvc.policyException).namespace }}
+  namespace: {{ .Values.pvc.policyException.namespace }}
```

Comment:
I think you forgot to add it to the values with a default. I think there are some restrictions at least on MCs where policy exceptions can be installed. Schema needs to be regenerated as well probably.

Reply:
Thinking about this: https://gigantic.slack.com/archives/C02FL4EAADD/p1706793331744919

Reply:
it should be fixed

Reply:
Thanks! LGMT! 👍🏻